addons/sale_loyalty/tests/test_sale_auto_invoice.py

```diff
@@ -0,0 +1,49 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.fields import Command
+from odoo.tests import tagged
+
+from odoo.addons.sale_loyalty.tests.common import TestSaleCouponCommon
+
+
+@tagged('post_install', '-at_install')
+class TestWebsiteSaleAutoInvoice(TestSaleCouponCommon):
+
+    def test_automatic_invoice_on_zero_amount_order(self):
+        # Set automatic invoice
```

Comment:
I would be better to add triple quoted docstring at start of function with step instead adding comment at every steps. It would look better IMO

But you can keep it as is if you like :wink: